src/Communication/docs/New-AzCommunicationServiceKey.md

```diff
@@ -0,0 +1,247 @@
+---
+external help file:
+Module Name: Az.Communication
+online version: https://docs.microsoft.com/en-us/powershell/module/az.communication/new-azcommunicationservicekey
+schema: 2.0.0
+---
+
+# New-AzCommunicationServiceKey
+
+## SYNOPSIS
+Regenerate CommunicationService access key.
+PrimaryKey and SecondaryKey cannot be regenerated at the same time.
+
+## SYNTAX
+
+### RegenerateExpanded (Default)
+```
+New-AzCommunicationServiceKey -CommunicationServiceName <String> -ResourceGroupName <String>
+ [-SubscriptionId <String>] [-KeyType <KeyType>] [-DefaultProfile <PSObject>] [-Confirm] [-WhatIf]
```

Comment:
>[-KeyType <KeyType>] [](start = 28, length = 20)

This parameter is required. The Swagger spec incorrectly says that it's optional, but I'm fixing that now.

Reply:
Cmdlet Directives are limited to renaming, aliases or removal. It's best IMO if we update the cmdlets to the swagger spec one that is updated to specify KeyType as a required parameter, rather than manually diverge the autogenerated files from the swagger.